.github/workflows/on_push.yaml

```diff
@@ -0,0 +1,13 @@
+on:
+  push:
+    branches:
+      - '*.*/*'
```

Comment:
shouldn't it be only push to `main`?

Reply:
We spoke about this offline, it was agreed to change the base branch to `8.0/edge` from `main` for this, and discuss later what the versioning/releasing strategy will be.